CRM/Contact/BAO/Contact.php

```diff
@@ -3720,6 +3720,10 @@ public static function getEntityRefFilters() {
       ['key' => 'tag', 'value' => ts('Tag'), 'entity' => 'EntityTag'],
       ['key' => 'state_province', 'value' => ts('State/Province'), 'entity' => 'Address'],
       ['key' => 'country', 'value' => ts('Country'), 'entity' => 'Address'],
+      ['key' => 'first_name', 'value' => ts('First Name'), 'type' => 'text', 'condition' => ['contact_type' => 'Individual']],
+      ['key' => 'last_name', 'value' => ts('Last Name'), 'type' => 'text', 'condition' => ['contact_type' => 'Individual']],
+      ['key' => 'nick_name', 'value' => ts('Nick Name'), 'type' => 'text', 'condition' => ['contact_type' => 'Individual']],
+      ['key' => 'organization_name', 'value' => ts('Employer name'), 'type' => 'text', 'condition' => ['contact_type' => 'Individual']],
```

Comment:
should the contact type be org here? or is it a different field we are searing in?

Reply:
@seamuslee001 check the PR comments - in this case it is called 'Employer name' & works for individuals. I figured not having it for orgs was OK as it basically IS the sort name that is searched anyway